### ISSUE TITLE
Break a part of the synchronization Between ol and cesium

### ISSUE THREAD

**Issue by LZS1991** [openlayers/ol3-cesium#422]
I want to add a layer that show both on the ol and the cesium, and i make it.But there is another layer that i just want to display in the ol without showing in the cesium, in which way can i achieve that?build another ol.map?

**Comment by gberaudo** [comment]
Unmanaged layers are not rendered in Cesium.
Another way is to create your own custom renders and only synchronize the layers you want: see `ol3cesium.js`: 
```
  var synchronizers = goog.isDef(options.createSynchronizers) ?
      options.createSynchronizers(this.map_, this.scene_, this.dataSourceCollection_) : [
        new olcs.RasterSynchronizer(this.map_, this.scene_),
        new olcs.VectorSynchronizer(this.map_, this.scene_)
      ];
```

**Comment by LZS1991** [comment]
I know that， and i try it in this way:
        var rasterSync = new olcs.RasterSynchronizer();
        var vectorSync = new olcs.VectorSynchronizer({
            synchronize: function() {
                // custom synchronize function
            }
        });
        olmap3d = new olcs.OLCesium({
            map: olmap2d,
            target: 'map3d',
            synchronizers: [rasterSync, vectorSync]
        });

but there is a exception:"scene is undefined".however,if i have not create a olcs.OLCesium,i can not get the scene,so how can i solve it?Thanks

**Comment by LZS1991** [comment]
Thanks,and can you give me an example that how i use the createSynchronizers?Such as the following:
        olmap3d = new olcs.OLCesium({
            map: olmap2d,
            target: 'map3d',
            createSynchronizers: createSync
        });

           function createSync(map, scene, dataSource) {
        // 
    }

Am i right?

**Comment by gberaudo** [comment]
Yes the createSynchronizers function should return an array of synchronizers. The snippet above shows how to instanciate the default synchonizers.

**Comment by LZS1991** [comment]
OK,I have done that ,but still one problem exists.when i want to add a vector layer,i use it in this way:
                var featureConverter = new olcs.FeatureConverter(scene);
                featureConverter.olVectorLayerToCesium(testLayer, map.getView(),scene.primitives);
                var vectorSync = new olcs.VectorSynchronizer(map, scene, featureConverter);
                vectorSync.synchronize();

I jsut add one vectorLayer,but in fact, when i do that,all the vectorlayers i add to the ol are added to the cesium.maybe it is my fault,and how to add just one vectorLayer to the cesium?

**Comment by gberaudo** [comment]
You can do something like:

```
goog.provide('olcs.MyVectorSynchronize');

/**
 * @param {!ol.Map} map
 * @param {!Cesium.Scene} scene
 * @param {olcs.FeatureConverter=} opt_converter
 * @constructor
 * @extends {olcs.VectorSynchronizer}
 * @api
 * @struct
 */
olcs.MyVectorSynchronizer = function(map, scene, opt_converter) {
  olcs.VectorSynchronizer.call(this, map, scene);
}
ol.inherits(olcs.MyVectorSynchronizer, olcs.VectorSynchronizer);

/**
 * @inheritDoc
 */
olcs.MyVectorSynchronizer.prototype.createSingleLayerCounterparts = function(olLayer) {
  if (mustSynchronizeToCesium(olLayer)) {
    return olcs.VectorSynchronizer.call(this, olLayer);
  } else {
   return null;
  }
}
```

Then you put this file in `src/plugins`. You compile `ol3-cesium` and you are able to use `olcs.MyVectorSynchronizer` in your application code.

**Comment by LZS1991** [comment]
Sorry for looking your answer now，I will try it！

**Comment by LZS1991** [comment]
I try it, but still something is wrong.
TypeError: map.getView is not a function 1 ol3cesium-debug.js:8752:15
	olcs.AbstractSynchronizer http://127.0.0.1/SWWebClient/modules/ol3-cesium/ol3cesium-debug.js:8752:15
	olcs.MyVectorSynchronizer.prototype.createSingleLayerCounterparts http://127.0.0.1/SWWebClient/modules/ol3-cesium/ol3cesium-debug.js:31255:16
I don't know how to deal with it.Can you help me?

**Comment by pointGH** [comment]
你的问题解决了吗

**Comment by measman** [comment]
Is your problem solved?
